Makefile.kokkos

```diff
@@ -24,33 +24,36 @@ KOKKOS_OPTIONS ?= ""
 # Options: force_uvm,use_ldg,rdc,enable_lambda
 KOKKOS_CUDA_OPTIONS ?= ""
 
+#TODO - I think this will work:
+ifhas=$(if $(findstring $2,$1),1,0)
```

Comment:
is it really necessary to replace how this is done? I know @crtrott had concerns, and the comment above this line is not encouraging.

Reply:
Sorry -- crufty comment.  It works.  Related to @crtrott comment above:  The `$(call ...)` functionality is supported everywhere other `$(...)` functionality is supported.  The use of `$(shell ...)` and `$(wildcard ...)` gave me confidence that I could use it.

In my personal testing, when I had a build going on my machine and bogging it down, the using of the internal shell functionality was noticably faster (removing ~80 calls out to a shell).

Reply:
> noticably faster (removing ~80 calls out to a shell)

okay, I guess this is a good reason to try to change this

Reply:
I'd propose a namechange of the function to 'kokkos_has_string'